src/unix/unix.c

```diff
@@ -12,6 +12,9 @@
 #define pf_debug(x, ...) thread_log(current, x, ##__VA_ARGS__);
 #endif
 
+#define MAX_PROCESSES 2
+static process processes[MAX_PROCESSES];
```

Comment:
I needed to remove the static here to run the example.

Reply:
Interesting--I wonder if gdb version or something affects this. I removed the static though so it should work now.